packages/atomic/src/components/atomic-result-list/atomic-result-list.tsx

```diff
@@ -90,6 +99,12 @@ export class AtomicResultList implements InitializableComponent {
    */
   @Prop() image: ResultDisplayImageSize = 'icon';
 
+  private renderingFunction?: (result: Result) => HTMLElement = undefined;
+
+  @Method() public setRenderFunction(render: (result: Result) => HTMLElement) {
```

Comment:
Should we set this as `@internal` ?

Reply:
Is this only for documentation ?

Reply:
Speaking of docs, a line to explain what this function is for would be useful, especially since it's not directly used inside Atomic

Reply:
Whether we make it public/documented or not, this will need to be maintained since we use it in atomic-react and it's likely going to be used in other frameworks. I have a slight preference for documenting it.

Reply:
@jpmarceau / @npushkarskii : Small review for the comment here please !

Reply:
@olamothe done! 😉